models/Element/AbstractElement.php

```diff
@@ -213,7 +213,7 @@ public function getId(): ?int
 
     public function setId(?int $id): static
     {
-        $this->id = $id ? (int)$id : null;
+        $this->id = $id ? $id : null;
```

Comment:
We could change this.
```suggestion
        $this->id = $id ?: null;
```